modules/cli/src/main/scala/scala/cli/commands/util/VerbosityOptionsUtil.scala

```diff
@@ -0,0 +1,12 @@
+package scala.cli.commands
+package util
+
+object VerbosityOptionsUtil {
+  implicit class VerbosityOptionsOps(v: VerbosityOptions) {
+    import v._
+
+    val Interactive = if (interactive.getOrElse(false))
```

Comment:
No need to use upper case here. By not importing `v._`, it should be possible to keep `interactive` as a name I think:
```suggestion
    val interactive = if (v.interactive.getOrElse(false))
```